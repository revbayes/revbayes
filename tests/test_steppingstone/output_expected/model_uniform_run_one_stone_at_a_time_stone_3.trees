Iteration	Posterior	Likelihood	Prior	psi
0	-21332.81	-21313.06	-19.75512	(Galeopterus_variegatus[&index=23]:0.286959,((((Colobus_guereza[&index=6]:0.203344,(Macaca_mulatta[&index=13]:0.142354,Chlorocebus_aethiops[&index=5]:0.175953)[&index=25]:0.075522)[&index=42]:0.182914,(Hylobates_lar[&index=9]:0.177057,Pan_paniscus[&index=17]:0.153385)[&index=32]:0.053928)[&index=30]:0.378576,(((Cebus_albifrons[&index=3]:0.164627,Aotus_trivirgatus[&index=1]:0.129312)[&index=26]:0.000909,Saimiri_sciureus[&index=20]:0.266103)[&index=33]:0.095955,Callicebus_donacophilus[&index=2]:0.121040)[&index=43]:0.278747)[&index=31]:0.222978,(Daubentonia_madagascariensis[&index=7]:0.311358,((Loris_tardigradus[&index=12]:0.189142,(Nycticebus_coucang[&index=15]:0.347662,(Perodicticus_potto[&index=18]:0.266928,(Galago_senegalensis[&index=8]:0.133561,Otolemur_crassicaudatus[&index=16]:0.113059)[&index=24]:0.090779)[&index=29]:0.044700)[&index=37]:0.028852)[&index=41]:0.078784,(((Lepilemur_hubbardorum[&index=11]:0.214327,Propithecus_coquereli[&index=19]:0.160518)[&index=39]:0.020793,(Lemur_catta[&index=10]:0.112913,Varecia_variegata_variegata[&index=22]:0.211138)[&index=38]:0.082882)[&index=40]:0.007929,(Cheirogaleus_major[&index=4]:0.080684,Microcebus_murinus[&index=14]:0.201808)[&index=35]:0.093787)[&index=27]:0.140677)[&index=34]:0.008790)[&index=28]:0.079386)[&index=36]:0.145080,Tarsius_syrichta[&index=21]:0.321850)[&index=44];
10	-21352.5	-21336.76	-15.7416	(Galeopterus_variegatus[&index=23]:0.387706,(Tarsius_syrichta[&index=21]:0.316362,(Daubentonia_madagascariensis[&index=7]:0.196975,((((Lemur_catta[&index=10]:0.128838,Varecia_variegata_variegata[&index=22]:0.221692)[&index=38]:0.059791,((Cheirogaleus_major[&index=4]:0.080902,Microcebus_murinus[&index=14]:0.220523)[&index=35]:0.110276,Propithecus_coquereli[&index=19]:0.229199)[&index=40]:0.006164)[&index=39]:0.020445,Lepilemur_hubbardorum[&index=11]:0.261840)[&index=27]:0.130405,((Perodicticus_potto[&index=18]:0.290401,(Otolemur_crassicaudatus[&index=16]:0.112730,Galago_senegalensis[&index=8]:0.102988)[&index=24]:0.073921)[&index=29]:0.043160,(Nycticebus_coucang[&index=15]:0.155927,Loris_tardigradus[&index=12]:0.262347)[&index=37]:0.036819)[&index=41]:0.070144)[&index=28]:0.065346)[&index=34]:0.028865)[&index=36]:0.072857,((Callicebus_donacophilus[&index=2]:0.091841,((Aotus_trivirgatus[&index=1]:0.075039,Cebus_albifrons[&index=3]:0.187795)[&index=26]:0.003593,Saimiri_sciureus[&index=20]:0.224832)[&index=33]:0.143432)[&index=43]:0.278298,((Hylobates_lar[&index=9]:0.101846,Pan_paniscus[&index=17]:0.207520)[&index=32]:0.013721,((Macaca_mulatta[&index=13]:0.160684,Chlorocebus_aethiops[&index=5]:0.158172)[&index=25]:0.043919,Colobus_guereza[&index=6]:0.196958)[&index=42]:0.097952)[&index=30]:0.321284)[&index=31]:0.226031)[&index=44];
