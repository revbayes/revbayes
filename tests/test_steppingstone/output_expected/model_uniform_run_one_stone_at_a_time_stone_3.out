state	power	likelihood
4	0.16807	-21390.4
5	0.16807	-21371.5
6	0.16807	-21371.2
7	0.16807	-21369.3
8	0.16807	-21337.9
9	0.16807	-21340.4
10	0.16807	-21370.3
